interface-definitions/include/dhcp/option-v4.xml.i

```diff
@@ -232,6 +232,25 @@
             </leafNode>
           </children>
         </node>
+        <node name="shoretel">
+          <properties>
+            <help>Shoretel-specfic parameters</help>
```

Comment:
I suppose we should consistently capitalize "ShoreTel" like they do in their branding.